### DISCUSSION TITLE
Monika not returning to the room after I put her character file back from taking her somewhere

### DISCUSSION THREAD

**Discussion by ChappyXx** [Monika-After-Story/MonikaModDev#9164] [QA - tech support + general] [answered]
Hi, im rlly hoping someone can help me. I’m currently dealing with a mas issue with puting monika’s file back after tsking her out but this time its not working.

I have do e this ttwice before and has been perfectly fine returning, but now when transfering her file into the characters file and opening the game it just gives me an empty room with an empty chair.

i have tried many methods, such as going into my bin and trying all the monika files that were deleted from past outings but this time they all have dates of times next to her name when it never did this before. Ive even tried loading a saved game from before only to come back to an empty room still.

Please any advice provided would be greatly appreciated, im a mac user if that helps too.

**Comment by ThePotatoGuy** [comment, marked as answer]
What appears in your [mfread log file](https://github.com/Monika-After-Story/MonikaModDev/wiki/Logs-and-Other-Files#mfread)?

Also you can [restore a persistent backup](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent) from before you took her out.
